test/unit/crate_name/main.rs

```diff
@@ -0,0 +1 @@
+fn main() {}
```

Comment:
FYI, you could use this file also for rust_library srcs, but I actually don't mind having a separate empty file, so it's up to you.

Reply:
I considered this too, but the rules read a bit weird with a `main.rs` in the sources of a `rust_library`, so I'd like to stay with this.